### DISCUSSION TITLE
Add tire pressure details and status of trunk/hood

### DISCUSSION THREAD

**Discussion by TiddlyWiddly** [bimmerconnected/bimmer_connected#483] [Feature request]
Hi,

Could you please add support for tire details, trunk, sunroof, and hood status? For tire details, the current and target pressure would be very nice to monitor as graphs in HomeAssistant and for the others just a simple open/close.  I don't know what tire wear status shows, but maybe that would be nice to monitor as well.

**Comment by rikroe** [comment]
Trunk, sunroof and hood status should be available in HomeAssistant, although only as attributes to the `lids`, `doors` and `windows` sensors. It is planned to change this at some point, but no ETA. If you want to have one of the values as separate sensor, you can use a template sensor on the characteristics.

I'll check how to best integrate tire pressure and wear status.

**Reply by TiddlyWiddly** [reply]
Thanks for taking a look and for the details about the lids. I never noticed it had attributes!

**Reply by TiddlyWiddly** [reply]
Hi I just wanted to check in again on tire state. If there’s any testing I can help with I’d be willing to do it.

**Comment by PuppetMaster2k** [comment]
Hello! Are there any plans on adding the "tireState"? Would be really nice to have them too.

**Reply by sincze** [reply]
It seems I am unable to find the lids and tyre pressure as well. However If I use the CLI version I do have this information available. Did I do something wrong in HA ??

**Reply by sincze** [reply]
@gerard33 It seems I am missing these in HomeAssistant as individual entities to work with. Or did I miss something?
![image](https://github.com/bimmerconnected/bimmer_connected/assets/5776333/783e0dad-e128-4f97-b1af-3fcf4eac8266)

Same goes for windows and lids. CLI has it. Homeassistant doesn't.
![image](https://github.com/bimmerconnected/bimmer_connected/assets/5776333/d9fba8c6-acb4-4808-9626-169eb6913f98)

To prevent API overload requests I use CLI now and HomeAssistant only once every hour. Would be nice if I could phase-out the CLI part ;-)

![image](https://github.com/bimmerconnected/bimmer_connected/assets/5776333/2cb5628f-704d-4f54-8473-3ee53883c2c6)

**Reply by rikroe** [reply]
I have created a PR that adds tire pressure to HA - see https://github.com/home-assistant/core/pull/120949.
